src/api_server.cc

```diff
@@ -159,21 +167,29 @@ void MetadataApiServer::HandleHealthz(
       LOG(INFO) << "/healthz returning 200";
     }
     conn->set_status(HttpServer::connection::ok);
+
+    std::string response = "healthy";
+
     conn->set_headers(std::map<std::string, std::string>({
       {"Content-Type", "text/plain"},
+      {"Content-Length", std::to_string(response.length())},
     }));
-    conn->write("healthy");
+    conn->write(response);
   } else {
     LOG(WARNING) << "/healthz returning 500; unhealthy components: "
                  << boost::algorithm::join(unhealthy_components, ", ");
     conn->set_status(HttpServer::connection::internal_server_error);
+
+    std::string response = "unhealthy components:\n";
+    for (const auto& component : unhealthy_components) {
+      response.append(component + "\n");
```

Comment:
Let's use `std::ostringstrean` instead of `std::string::append`.

Reply:
Updated, but I'll link to this for context:
https://stackoverflow.com/questions/905479/stdstring-length-and-size-member-functions